### PR TITLE
added anthem translation field as of enhancement issue #21

### PR DESCRIPTION
**Pull request by MonEstCha**

### CODE REVIEW

**Comment by jftsang** [comment]
added you as a collaborator

File: forms.py [hunk 1]
```diff
@@ -38,6 +42,7 @@ class PewSheetForm(FlaskForm):
     anthem_title = StringField('Anthem')
     anthem_composer = StringField('Anthem composer')
     anthem_lyrics = StringField('Anthem lyrics', widget=TextArea())
+    anthem_translation = SelectField('Anthem Translation', choices=translations)
```
Comment (jftsang) [inline forms.py:45]: I don't know if Flask allows nesting of forms but it might be nice to group these together

File: models.py [hunk 2]
```diff
@@ -438,13 +440,14 @@ def from_form(cls, form: 'PewSheetForm') -> 'Service':
         else:
             secondary_feasts = []
 
-        if form.anthem_title.data or form.anthem_composer.data or form.anthem_lyrics.data:
+        if form.anthem_title.data or form.anthem_composer.data or form.anthem_lyrics.data or form.anthem_translation.data:
```
Comment (jftsang) [inline models.py:443]: this would be simplified if we could group stuff together

Comment (MonEstCha) [reply models.py:443]: Okay, I will do some configurations in my git repo to have the commits for the two different issues on two different branches and then I will try to group the anthem fields together.

**Comment by MonEstCha** [comment]
@jftsang, meanwhile I'm trying to group the anthem fields, can you give me any hint on how to fix the error with the bad request?

File: models.py (hunk 2 above)
Comment (MonEstCha) [reply models.py:443]: Okay, I will give it a try!

**Comment by MonEstCha** [comment]
@jftsang, could you review my pull request, please?

**Pull request merged**